apps/server/src/controllers/userController.ts

```diff
@@ -104,3 +107,53 @@ export const registerUser = async (req: Request, res: Response) => {
     })
   }
 }
+
+export const forgotPassword = async (req: Request, res: Response) => {
+  try {
+    const mailSchema = z.object({
+      email: z.string().email({ message: 'email is invalid' }),
+    })
+    const validate = mailSchema.strict().safeParse(req.body)
+
+    if (validate.success) {
+      let { email } = req.body
+      email = email.trim().toLowerCase()
+      const user = await Users.findOne({ where: { email: email } })
+      const passwordResetData = GenerateOTP()
+      const passwordResetToken = passwordResetData.otp.toString()
+      const tokenExpiery = passwordResetData.expiry
```

Comment:
You may want to correctly spell the variable in line 124(tokenExpiry)

Reply:
I have updated the code